src/provisioner/provisioner.py

```diff
@@ -247,6 +247,46 @@ def cluster_bootstrap(cortx_conf_url: str, force_override: bool = False):
         """
         Description:
 
+        Configures Cluster Components
+        1. Compares current installed version with New version
+        2. Invoke Mini Provisioners of cluster components deploy/upgrade based on version compatibility
+        Paramaters:
+        [IN] CORTX Config URL
+        """
+        cortx_conf = MappedConf(cortx_conf_url)
+        node_id = Conf.machine_id
+        Conf.load(const.TMP_IDX, const.CORTX_TMP_URL)
```

Comment:
Same here. Naming of CORTX_TMP_URL, e.g. CORTX_CONF_UR (preferred)L or TMP_CORTX_CONF_URL

Reply:
done